packages/opentelemetry-instrumentation-crewai/opentelemetry/instrumentation/crewai/instrumentation.py

```diff
@@ -0,0 +1,130 @@
+    
+    def _uninstrument(self, **kwargs):
+        pass
+
+
+def with_tracer_wrapper(func):
+    """Helper for providing tracer for wrapper functions."""
+
+    def _with_tracer(tracer):
+        def wrapper(wrapped, instance, args, kwargs):
+            return func(tracer, wrapped, instance, args, kwargs)
+        return wrapper
+    return _with_tracer
+
+
+@with_tracer_wrapper
+def wrap_kickoff(tracer, wrapped, instance, args, kwargs):
+    with tracer.start_as_current_span(
+        "crewai.workflow",
+        kind=SpanKind.INTERNAL,
+        attributes={
+        }
+    ) as span:
+        try:
+            CrewAISpanAttributes(span=span, instance=instance)
+            result = wrapped(*args, **kwargs)
+            if result:
+                class_name = instance.__class__.__name__
+                span.set_attribute(f"crewai.{class_name.lower()}.result", str(result))
+                span.set_status(Status(StatusCode.OK))
+                if class_name == "Crew":
+                    for attr in ["tasks_output", "token_usage", "usage_metrics"]:
+                        if hasattr(result, attr):
+                            span.set_attribute(f"crewai.crew.{attr}", str(getattr(result, attr))) 
+            return result
+        except Exception as ex:
+            span.set_status(Status(StatusCode.ERROR, str(ex)))
+            raise
+
+
+@with_tracer_wrapper
+def wrap_agent_execute_task(tracer, wrapped, instance, args, kwargs):
+    agent_name = instance.role if hasattr(instance, "role") else "agent"
+    with tracer.start_as_current_span(
+        f"{agent_name}.agent",
+        kind=SpanKind.CLIENT,
+        attributes={
+            SpanAttributes.TRACELOOP_SPAN_KIND: TraceloopSpanKindValues.AGENT.value,
+        }
+    ) as span:
+        try:
+            CrewAISpanAttributes(span=span, instance=instance)
+            result = wrapped(*args, **kwargs)
+            class_name = instance.__class__.__name__
+            span.set_attribute(f"crewai.{class_name.lower()}.model", str(instance.llm.model))
+            span.set_status(Status(StatusCode.OK))
+            return result
+        except Exception as ex:
+            span.set_status(Status(StatusCode.ERROR, str(ex)))
+            raise
+        
+@with_tracer_wrapper
+def wrap_task_execute(tracer, wrapped, instance, args, kwargs):
+    task_name = instance.description if hasattr(instance, "description") else "task"
+
+    with tracer.start_as_current_span(
+        f"Task.execute.{task_name}",
+        kind=SpanKind.CLIENT,
+        attributes={
+            SpanAttributes.TRACELOOP_SPAN_KIND: TraceloopSpanKindValues.TASK.value,
+        }
+    ) as span:
+        try:
+            CrewAISpanAttributes(span=span, instance=instance)
+            result = wrapped(*args, **kwargs)
+            class_name = instance.__class__.__name__
+            span.set_attribute(f"crewai.{class_name.lower()}.result", str(result))
```

Comment:
this should be in the output attribute:
https://github.com/traceloop/openllmetry/blob/36dd9b3c89dceaf50d754d82b25c5eb53af567e2/packages/opentelemetry-instrumentation-langchain/opentelemetry/instrumentation/langchain/callback_handler.py#L471